simulators/trin-bridge/src/main.rs

```diff
@@ -1,17 +1,73 @@
+mod constants;
+
+use crate::constants::{
+    BERLIN_BLOCK_NUMBER, BOOTNODES_ENVIRONMENT_VARIABLE, BYZANTIUM_BLOCK_NUMBER,
+    CONSTANTINOPLE_BLOCK_NUMBER, HIVE_CHECK_LIVE_PORT, HOMESTEAD_BLOCK_NUMBER,
+    ISTANBUL_BLOCK_NUMBER, LONDON_BLOCK_NUMBER, MERGE_BLOCK_NUMBER, SHANGHAI_BLOCK_NUMBER,
+    TEST_DATA_FILE_PATH, TRIN_BRIDGE_CLIENT_TYPE,
+};
 use ethportal_api::HistoryContentKey;
 use ethportal_api::HistoryContentValue;
 use ethportal_api::PossibleHistoryContentValue;
 use ethportal_api::{Discv5ApiClient, HistoryNetworkApiClient};
+use hivesim::types::ClientDefinition;
 use hivesim::{dyn_async, Client, Simulation, Suite, Test, TestSpec, TwoClientTestSpec};
 use itertools::Itertools;
-use portal_bridge::bridge::Bridge;
-use portal_bridge::execution_api::ExecutionApi;
-use portal_bridge::mode::BridgeMode;
-use portal_bridge::pandaops::PandaOpsMiddleware;
 use serde_yaml::Value;
+use std::collections::HashMap;
 use tokio::time::Duration;
-use trin_validation::accumulator::MasterAccumulator;
-use trin_validation::oracle::HeaderOracle;
+
+fn get_flair(block_number: u64) -> String {
```

Comment:
Same with this util, it looks to be exactly the same as the `get_flair` in `portal-interop/src/main.rs`, I think it's worthwhile to find a single home in the codebase for this util

Reply:
Don't think this question was resolved?